components/telemetry-operator/webhooks/logpipeline/webhook_suite_test.go

```diff
@@ -126,20 +126,20 @@ var _ = BeforeSuite(func() {
 
 	logPipelineValidator := NewValidatingWebhookHandler(mgr.GetClient(), inputValidatorMock, variableValidatorMock, pluginValidatorMock, maxPipelinesValidatorMock, outputValidatorMock, fileValidatorMock, dryRunnerMock)
 
-	By("registering LogPipeline webhook")
+	By("registering LogPipeline webhooks")
 	mgr.GetWebhookServer().Register(
 		"/validate-logpipeline",
 		&k8sWebhook.Admission{Handler: logPipelineValidator})
 
-	//+kubebuilder:scaffold:webhook
+	//+kubebuilder:scaffold:webhooks
```

Comment:
I think in this file, renaming went wrong and also renamed all comments with occurences of `webhook` to `webhooks` everywhere